src/memgraph.cpp

```diff
@@ -1200,20 +1204,35 @@ int main(int argc, char **argv) {
                             []() -> nlohmann::json { return query::plan::CallProcedure::GetAndResetCounters(); });
   }
 
+  communication::websocket::Server websocket_server{
+      boost::asio::ip::tcp::endpoint{boost::asio::ip::make_address("0.0.0.0"), 7444}};
+
+  {
+    auto sinks = spdlog::default_logger()->sinks();
+    sinks.push_back(websocket_server.GetLoggingSink());
+    spdlog::set_default_logger(std::make_shared<spdlog::logger>("memgraph_log", sinks.begin(), sinks.end()));
+  }
+
   // Handler for regular termination signals
-  auto shutdown = [&server, &interpreter_context] {
+  auto shutdown = [&websocket_server, &server, &interpreter_context] {
     // Server needs to be shutdown first and then the database. This prevents
     // a race condition when a transaction is accepted during server shutdown.
     server.Shutdown();
     // After the server is notified to stop accepting and processing
     // connections we tell the execution engine to stop processing all pending
     // queries.
     query::Shutdown(&interpreter_context);
+    websocket_server.Shutdown();
```

Comment:
Shouldn't we wait for the server to shut down properly? Or why not stop and wait for the server in the destructor of `websoket::Server`?

Reply:
We are waiting for it bellow (`websocket_server.AwaitShutdown()`)
I went with the same solution as for the Bolt Server, so that's why we don't wait in the destructor.